src/main/java/tc/oc/pgm/commands/RotationCommands.java

```diff
@@ -0,0 +1,269 @@
+package tc.oc.pgm.commands;
+
+import app.ashcon.intake.Command;
+import app.ashcon.intake.CommandException;
+import app.ashcon.intake.parametric.annotation.Default;
+import java.util.List;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.command.CommandSender;
+import tc.oc.pgm.AllTranslations;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.chat.Audience;
+import tc.oc.pgm.api.match.MatchManager;
+import tc.oc.pgm.map.PGMMap;
+import tc.oc.pgm.rotation.FixedPGMMapOrder;
+import tc.oc.pgm.rotation.FixedPGMMapOrderManager;
+import tc.oc.pgm.util.PrettyPaginatedResult;
+
+public class RotationCommands {
+  @Command(
+      aliases = {"rotation", "rot"},
+      desc = "Shows the maps in the active rotation",
+      usage = "[page]",
+      help = "Shows all the maps that are currently in the active rotation.")
+  public static void rotation(
+      Audience audience, CommandSender sender, MatchManager matchManager, @Default("1") int page)
+      throws CommandException {
+
+    if (matchManager.getMapOrder() instanceof FixedPGMMapOrderManager) {
+      FixedPGMMapOrderManager fixedPGMMapOrderManager =
+          (FixedPGMMapOrderManager) matchManager.getMapOrder();
+      FixedPGMMapOrder rotation = fixedPGMMapOrderManager.getActiveRotation();
+
+      List<PGMMap> maps;
+      if (rotation != null) maps = rotation.getMaps();
+      else {
+        sender.sendMessage(
+            ChatColor.RED + AllTranslations.get().translate("command.rotation.noRotation", sender));
+        return;
+      }
+
+      int resultsPerPage = 8;
+      int pages = (maps.size() + resultsPerPage - 1) / resultsPerPage;
+
+      String listHeader =
+          ChatColor.BLUE.toString()
+              + ChatColor.STRIKETHROUGH
+              + "-----------"
+              + ChatColor.RESET
+              + " "
+              + AllTranslations.get().translate("command.rotation.currentRotation.title", sender)
+              + ChatColor.DARK_AQUA
+              + " ("
+              + ChatColor.AQUA
+              + rotation.getName()
+              + ChatColor.DARK_AQUA
+              + ")"
+              + " ("
+              + ChatColor.AQUA
+              + page
+              + ChatColor.DARK_AQUA
+              + " of "
+              + ChatColor.AQUA
+              + pages
+              + ChatColor.DARK_AQUA
+              + ") "
+              + ChatColor.translateAlternateColorCodes('&', "&9&m-----------");
```

Comment:
TBH, there is probably a simpler way to do this, but I'm okay with this for now.

Reply:
;_;